source/model.py

```diff
@@ -697,6 +697,7 @@ def _build_inclusions(start_dir, prefix, recursive):
                         p.add_path(path)
 
                     p.on_include = Header._on_include
+                    p.all_inclusions = True
```

Comment:
We need a mechanism to force `test_commits` to rebuild the cache at particular commit.
What about a `#RebuildQVC` hashtag in commit message. Case insensitive, likely.

Reply:
[related](https://github.com/ispras/qdt/pull/11#issuecomment-559774915)